Formula/nushell.rb

```diff
@@ -33,7 +33,24 @@ class Nushell < Formula
   end
 
   def install
+    puts "-----------------------------------------------------------------"
+    puts "Installing nushell (nu) with --features=extra and all the plugins"
+    puts "-----------------------------------------------------------------"
+    puts
+
+    puts "Install nushell from local..."
+    puts "----------------------------------------------"
     system "cargo", "install", "--features", "extra", *std_cargo_args
+
+    %w[nu_plugin_inc nu_plugin_gstat nu_plugin_query nu_plugin_example].each do |plugin|
+      puts
+      puts "----------------------------------------------"
+      puts "Install plugin #{plugin} from local..."
+      puts "----------------------------------------------"
+      cd "crates/#{plugin}" do
+        system "cargo", "install", *std_cargo_args
+      end
```

Comment:
```suggestion
    buildpath.glob("crates/nu_plugin_*").each do |plugindir|
      next unless (plugindir/"Cargo.toml").exist?

      system "cargo", "install", *std_cargo_args(path: plugindir)
```

Reply:
Things would be much easier if you told be earlier, Anyway, Thanks. I will update it soon